src/QMCHamiltonians/ECPComponentBuilder.h

```diff
@@ -33,6 +34,8 @@ struct ECPComponentBuilder : public MPIObjectBase, public QMCTraits
 
   int NumNonLocal;
   int Lmax, Llocal, Nrule;
+  int NumSO;
```

Comment:
We need documentation when things are updated or added.

Reply:
Done.